packages/flight-icons/catalog.json

```diff
@@ -9461,6 +9605,24 @@
       "width": 16,
       "height": 16
     },
+    {
+      "id": "5087:302",
+      "fileName": "transform-24",
```

Comment:
A general consideration (not blocker, but something to evaluate: the name of the icon `transform` is very generic (like `edit`) but the icon visualization/glyph is not: from the look of it, seems connected  to something more specific/a functionality of one of our products. Do you think it would be possible to rename the icon so it's more specialized – eg. `transform-[something]`– in case in the future we need to introduce a generic `transform` icon?

Reply:
That's a good point. Let me brainstorm with the Vault team on this.

Reply:
We've decided to change this to `transform-data`. I will get that update made today and re-request review.